src/main/webapp/js/pages/planner.js

```diff
@@ -28,9 +28,15 @@ async function getPlan() {
     selectedClasses: selectedClasses,
     semesters: document.getElementById('semesters').value
   };
-  courseContainer.innerText = '';  // Clears card body to get rid of spinner
+  courseContainer.innerText = '';  // Clears card body to get rid of spinner.
   let response;
   let courseList;
+  // Clear save button from header.
+  document.getElementById('plan-header').innerText = '';
+  // Replace form to clear all event listeners so it doesn't save old schedules.
```

Comment:
not the cleanest approach (you could clear the event listeners manually instead), but in the interest of time, this will work.